### ISSUE TITLE
Upgrading UDM Pro - Boot Script

### ISSUE THREAD

**Issue by Jiriteach** [boostchicken/udm-utilities#246]
Hi - Awesome work on these. They work really well. Am using the boot script for policy based routing so I can effectively use dual WAN's.

My UDM Pro is set to auto-update on the early access channel. When this happens - what will happen to the boot script and do I need to reinstall or will it just work?

Thanks

**Comment by alloylab** [comment]
@Jiriteach can you share your policy based routing script? I would be interested to see what you have done with dual wans.. thanks.

**Comment by mabunixda** [comment]
@Jiriteach the recent updates - i do not want to auto update and just read the changes before ( and wait some others to upgrade ) - did not request an reinstall on my udmp

**Comment by Jiriteach** [comment]
So anyone know if anything special is required when the UDM Pro's update when running the boot script? Not that my UDM Pro ever reboots so not sure its worth running the boot script anyways.

**Comment by jperquin** [comment]
Sorry to revive this thread but my issue appears closely related..

- UDM-Pro with on_boot installed (and running AdguardHome succesfully)
- WAN1 on eth9
- U-LTE Pro EU with sim card providing a fixed external ip that is connected to the UDM-P on 192.168.1.145
- Running a couple of hosts on VLAN 50 (subnet 192.168.50.0/24) with ports forwarded:
   - 80, 81 & 443 to 192.168.50.36
   - 1883 to 192.168.50.101

When LTE fail-over kicks in my DDNS updates a number of hosts to the fixed ip address of my mobile provider, but by default the UDM-P does NOT update the port forwards to the LTE interface, so the hosts are not reachable from the outside (which is the whole point of the LTE fail-over setup, in my case..).

Was hoping someone could help me with a suitable on_boot script that would permanently forward any incoming traffic on the LTE to the correct hosts / ports..

Thanks in advance, Joost

**Comment by Jiriteach** [comment]
I have my port forwards for each WAN set on the UDM Pro. 

So 443 WAN1 and 443 WAN2 and so it's running in a dual WAN mode. 

When WAN1 is down - UDM Pro fails over to WAN2 and all works.

**Comment by jperquin** [comment]
Thanks Jithen.

Not sure if your setup actually utilizes the unifi branded LTE modem.. that’s where my troubles originate..

**Comment by Jiriteach** [comment]
Yup - I have a feeling that the UDM Pro handles that Ubiquiti LTE device different.

I'm using a seperate LTE router so the UDM Pro just thinks it's a DHCP or static IP uplink.

**Comment by jperquin** [comment]
Indeed. I have a Tetonika RUT240 LTE router in bridge mode that can provide a direct fixed IP to WAN2..

I’d like to be able to only give the subnet 192.168.50.1/24 on VLAN 50 access to WAN2 when fail-over kicks in.

what routing should I enable to block any of the other subnets / VLANs from using WAN2?

**Comment by Jiriteach** [comment]
Have a look at my script. I route a whole seperate subnet through my LTE connection. This is a seperate SSID on VLAN 20 but the concept is the same.
